lib/mix/phoenix/context.ex

```diff
@@ -25,12 +25,12 @@ defmodule Mix.Phoenix.Context do
     ctx_app   = opts[:context_app] || Mix.Phoenix.context_app()
     base      = Module.concat([Mix.Phoenix.context_base(ctx_app)])
     module    = Module.concat(base, context_name)
-    alias     = module |> Module.split() |> tl() |> Module.concat()
-    basename  = Phoenix.Naming.underscore(context_name)
-    dir       = Mix.Phoenix.context_lib_path(ctx_app, basename)
+    alias     = module |> Module.split() |> List.last() |> Module.concat(nil)
```

Comment:
Without them, the alias for a nested context like `Site.Blog` would be `Site.Blog`, and for what I've read it should be only `Blog`, but tell me if I'm wrong 😊. 

I took as an example the code on `lib/mix/phoenix/schema.ex`
```elixir
%Schema{
...
alias: module |> Module.split() |> List.last() |> Module.concat(nil),
...
```